en/pl/jsp.rst

```diff
@@ -137,30 +130,46 @@ You can refer to the :ref:`jsp-load-java`.
 
     % loadjava demodb SpCubrid.class
 
+Registering Stored Functions/Procedures
+=======================================
 
-Publish the loaded Java class
------------------------------
+CUBRID requires a process to register (publish) Java classes so that Java methods can be called from SQL statements or Java applications.
```

Comment:
```suggestion
CUBRID requires a process to register (publish) Java classes so that Java Stored Functions/Procedures can be called from SQL statements or applications.
```

Reply:
method라는 용어 대신 stored function/procedure 로 통일해서 사용하면 좋겠네요. 그리고 Java application으로 국한 된 것처럼 느껴져서 java는 제거합니다.

"한글 매뉴얼에도 반영해주세요. !!!"